packages/thirdweb/src/utils/extensions/drops/get-set-claim-condition-phases.ts

```diff
@@ -0,0 +1,74 @@
+import { maxUint256 } from "viem";
+import { NATIVE_TOKEN_ADDRESS } from "../../../constants/addresses.js";
+import type { ThirdwebContract } from "../../../contract/contract.js";
+import type { SetClaimConditionsParams as GeneratedParams } from "../../../extensions/erc1155/__generated__/IDrop1155/write/setClaimConditions.js";
+import { upload } from "../../../storage/upload.js";
+import { dateToSeconds } from "../../date.js";
+import { toHex } from "../../encoding/hex.js";
+import { convertErc20Amount } from "../convert-erc20-amount.js";
+import { processOverrideList } from "./process-override-list.js";
+import type { ClaimConditionsInput } from "./types.js";
+
+/**
+ * @param options
+ * @utils
+ */
+export async function getSetClaimConditionPhases(options: {
```

Comment:
this name is... very confusing.

I should be able to get a good idea of what this is for just by looking at the name.

Also i really dont think we should be exposing this this way. The inputs are kinda internal to us, you need to understand what tokenDecimals is for, what resetClaimEligibility does etc.

And once we expose it publicly, we can't change the API anymore.

Remind me again what's the use case you're trying to solve here? will help us figure out a good public API

Reply:
I can do better here. Let me update the code and ping you again. This is a non-urgent issue